app/controllers/homes_controller.rb

```diff
@@ -3,7 +3,9 @@ def show
     if signed_out?
       render "pages/landing"
     else
-      @recommended_items = RecommendedItem.where(user: current_user.with_following)
+      @recommended_items = RecommendedItem.
```

Comment:
`RecommendedItem.where(user:  current_user.with_following)` scopes it to the current user and their followers.

`current_user.recommended_items` would get multiple recommended items from one user. Instead we approach from the other side of the association so that we can select multiple recommended items by multiple users.